docs/bagit/readme.txt

```diff
@@ -0,0 +1,51 @@
+This zip file contains a HydroShare resource in Bagit format.  
```

Comment:
@pkdash I'll defer this question to @horsburgh and @dtarb: Do we want to list this readme.txt file in the bag structure tree content inside this readme.txt file?